tests/test_h1_smart_device.py

```diff
@@ -0,0 +1,91 @@
+import pytest
```

Comment:
Nits:

- missing `# uses the psf/black style` at the first line;
- for consistency, I prefer the filename `test_nzxt_h1_v2.py`, even if it's uglier.

Reply:
Doh missed copy/paste on that one :D